_quarto.yml

```diff
@@ -49,7 +49,7 @@ website:
     giscus:
       # Reference: https://quarto.org/docs/reference/projects/books.html#giscus
       # Also https://giscus.app/ for all options
-      repo: RConsortium/asa-biop-swe-wg
+      repo: openstatsware/website
       repo-id: "R_kgDOIEo72A"
```

Comment:
I guess these IDs can stay? Or do they also need to be updated?

Reply:
Good question! Where did you get the `repo-id` when you setup Giscus for the first time?

Reply:
On https://giscus.app/

Reply:
Thanks – checked there and looks like it is all setup properly! 😃 (IDs are correct)